src/plugins/camera_tracking/CameraTracking.cc

```diff
@@ -208,12 +219,19 @@ void CameraTrackingPrivate::Initialize()
   gzmsg << "Camera pose topic advertised on ["
          << this->cameraPoseTopic << "]" << std::endl;
 
-   // follow offset
-   this->followOffsetService = "/gui/follow/offset";
-   this->node.Advertise(this->followOffsetService,
-       &CameraTrackingPrivate::OnFollowOffset, this);
-   gzmsg << "Follow offset service on ["
-          << this->followOffsetService << "]" << std::endl;
+  // follow offset
+  this->followOffsetService = "/gui/follow/offset";
+  this->node.Advertise(this->followOffsetService,
+      &CameraTrackingPrivate::OnFollowOffset, this);
+  gzmsg << "Follow offset service on ["
+         << this->followOffsetService << "]" << std::endl;
+
+  // follow pgain
+  this->followPGainService = "/gui/follow/pgain";
```

Comment:
suggestion to change `pgain` to `p_gain`, i.e. `/gui/follow/p_gain`, to be consistent with other places in gz that use this term, e.g. https://github.com/gazebosim/gz-sim/blob/gz-sim7/src/systems/joint_controller/JointController.cc#L126

Reply:
Changed it everywhere.